src/main/java/at/hannibal2/skyhanni/features/garden/composter/ComposterOverlay.kt

```diff
@@ -494,8 +494,7 @@ object ComposterOverlay {
             HypixelCommands.getFromSacks(internalName.asString(), itemsNeeded - havingInInventory)
             // TODO Add sack type repo data
 
-            val isDwarvenMineable =
-                internalName.let { it.equals("VOLTA") || it.equals("OIL_BARREL") || it.equals("BIOFUEL") }
+            val isDwarvenMineable = internalName.let { it.equals("VOLTA") || it.equals("OIL_BARREL") || it.equals("BIOFUEL") }
```

Comment:
Reformat code still says this is wrong

Reply:
It's a really short line.
There's much longer lines in this file like 619
I think it's much cleaner this way, but ig I can change it

Reply:
those lines are not longer

Reply:
oh wait yeah